databases.go

```diff
@@ -120,6 +147,12 @@ type ClusterPrice struct {
 	Monthly float32 `json:"monthly"`
 }
 
+// DatabaseFork describes the source and restore time for the fork for forked DBs
+type DatabaseFork struct {
+	Source      int        `json:"source"`
+	RestoreTime *time.Time `json:"restore_time"`
+}
```

Comment:
Could `RestoreTime` be made omitempty here?

Reply:
Fixed